config/software/ruby.rb

```diff
@@ -208,10 +199,6 @@
     # need to patch ruby's configure file so it knows how to find shared libraries
     patch source: "ruby-aix-configure_26_and_later.patch", plevel: 1, env: patch_env
 
-    if version.satisfies?("~> 2.6.4")
-      patch source: "ruby-2.6.4-bug14834.patch", plevel: 1, env: patch_env
-    end
```

Comment:
that one is all of 2.6 isn't it?

Reply:
According to that bug it was backported 2 years ago so I don't think we need this one anymore